.github/workflows/build-hpc.yml

```diff
@@ -55,6 +55,7 @@ jobs:
             modules: ~
             build_options: --with-loki=ON --with-field-api=ON
             # TODO: These tests trigger a SIGFPE, to be investigated
+            #  'dwarf-cloudsc-gpu-std-par-scc-k-caching-dp-serial' works with NVHPC 22.11 but not with 24.5
```

Comment:
Is the failure with 24.5 also a SIGFPE or something different? Would be good to be explicit here.